python/generational/decisions.py

```diff
@@ -117,9 +120,9 @@ def __init__(self, id):
         self.threshold_reward = 0                              # used to scale fitness
         ## Action Tables
         self.target_bias = {
-            "working": {[self.satisfaction_score, self.energy_score, self.money_score], 1},
-            "eating": {[self.satisfaction_score, self.energy_score, self.money_score], 1},
-            "mating": {[self.satisfaction_score, self.energy_score, self.money_score], 1}
+            "working": ((self.satisfaction_score, self.energy_score, self.money_score), 1),
```

Comment:
{} is for dictionary or set, I'm assuming you want to just store those values, so I changed to tuples, but if those value changes, then you may want to use list instead -- tuples are not mutable

Reply:
I'm not sure if I want to use an enum, dictionary or a list yet, but it needs to be mutable. I will test a few ideas out.